crates/reth/primitives/src/lib.rs

```diff
@@ -13,7 +13,8 @@ pub struct WithdrawalIntent {
     pub amt: u64,
 
     /// Destination public key for the withdrawal
-    pub dest_pk: B256,
+    // FIXME: How do I use a BOSD Descriptor here?
+    pub destination: B256,
```

Comment:
`Vec<u8>` still, we can avoid validating it until we actually use it and take advantage of the simpler representation elsewhere.